src/main/scala/io/iohk/ethereum/domain/Account.scala

```diff
@@ -1,21 +1,26 @@
 package io.iohk.ethereum.domain
 
 import akka.util.ByteString
-import org.spongycastle.util.encoders.Hex
+
+object Account {
+  val Empty = Account(0, 0, ByteString.empty, ByteString.empty)
```

Comment:
there are hashes for empty storage root and empty evm code
https://github.com/input-output-hk/etc-client/blob/phase/3/blockSync/src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncNodesRequestHandler.scala#L126